ydb/core/tx/datashard/change_collector_async_index.cpp

```diff
@@ -105,7 +105,7 @@ bool TAsyncIndexChangeCollector::Collect(const TTableId& tableId, ERowOp rop,
     const auto tagToPos = MakeTagToPos(tagsToSelect, [](const auto tag) { return tag; });
     const auto updatedTagToPos = MakeTagToPos(updates, [](const TUpdateOp& op) { return op.Tag; });
 
-    userTable->ForEachAsyncIndex([&] (const auto& pathId, const auto& index) {
+    userTable->ForEachAsyncIndex([&](const auto& pathId, const TUserTable::TTableIndex& index) {
```

Comment:
I changed `auto` to `TTableIndex`, because otherwise vscode cannot find usage of `TTableIndex` members